TeslaApp/TeslaApp/CustomSlider/CustomSlider.swift

```diff
@@ -0,0 +1,78 @@
+// CustomSlider.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Кастомный слайдер
+struct CustomSlider<Component: View>: View {
+    // MARK: - Public Properties
+
+    @Binding var value: Double
+
+    var body: some View {
+        GeometryReader { geometry in
+            self.view(geometry: geometry)
+        }
+    }
+
+    // MARK: - Private Properties
+
+    private var range: (Double, Double)
+    private var knobWidth: CGFloat?
+    private let viewBuilder: (CustomSliderComponents) -> Component
```

Comment:
```suggestion
    private let viewBuilder: (CustomSliderComponents) -> Component
```
поставь выше переменных